src/main/java/com/smartsheet/api/internal/SheetRowResourcesImpl.java

```diff
@@ -86,49 +84,42 @@ public SheetRowResourcesImpl(SmartsheetImpl smartsheet) {
+     * @throws IllegalArgumentException : if any argument is null
+     * @throws InvalidRequestException : if there is any problem with the REST API request
+     * @throws AuthorizationException : if there is any problem with the REST API authorization(access token)
+     * @throws ResourceNotFoundException : if the resource can not be found
+     * @throws ServiceUnavailableException : if the REST API service is not available (possibly due to rate limiting)
+     * @throws SmartsheetRestException : if there is any other REST API related error occurred during the operation
+     * @throws SmartsheetException : if there is any other error occurred during the operation
      */
     public List<Row> addRows(long sheetId, List<Row> rows) throws SmartsheetException {
         return this.addRows(sheetId, rows, null, null);
     }
 
     /**
      * Insert rows to a sheet.
-     *
+     * <p>
      * It mirrors to the following Smartsheet REST API method: POST /sheets/{sheetId}/rows
-     *
-     * Exceptions:
-     *   - IllegalArgumentException : if any argument is null
-     *   - InvalidRequestException : if there is any problem with the REST API request
-     *   - AuthorizationException : if there is any problem with the REST API authorization(access token)
-     *   - ResourceNotFoundException : if the resource can not be found
-     *   - ServiceUnavailableException : if the REST API service is not available (possibly due to rate limiting)
-     *   - SmartsheetRestException : if there is any other REST API related error occurred during the operation
-     *   - SmartsheetException : if there is any other error occurred during the operation
-     *
      * @param sheetId the sheet id
      * @param rows the list of rows to create
      * @param includes optional objects to include
      * @param excludes optional objects to exclude
      * @return the created rows
-     * @throws SmartsheetException the smartsheet exception
+     * @throws IllegalArgumentException : if any argument is null
+     * @throws InvalidRequestException : if there is any problem with the REST API request
+     * @throws AuthorizationException : if there is any problem with the REST API authorization(access token)
+     * @throws ResourceNotFoundException : if the resource can not be found
+     * @throws ServiceUnavailableException : if the REST API service is not available (possibly due to rate limiting)
+     * @throws SmartsheetRestException : if there is any other REST API related error occurred during the operation
+     * @throws SmartsheetException : if there is any other error occurred during the operation
      */
-    public List<Row> addRows(long sheetId, List<Row> rows, EnumSet<RowInclusion> includes, EnumSet<ObjectExclusion> excludes) throws SmartsheetException {
+    public List<Row> addRows(long sheetId, List<Row> rows, EnumSet<RowInclusion> includes, EnumSet<ObjectExclusion> excludes)
+            throws SmartsheetException {
```

Comment:
This should probably be 
```suggestion
    public List<Row> addRows(
        long sheetId, 
        List<Row> rows,
        EnumSet<RowInclusion> includes,
        EnumSet<ObjectExclusion> excludes
    ) throws SmartsheetException {
```

Reply:
Agreed. Fixed